workspaces/simon-game/src/app/components/Box.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+type Props = { color?: string; height?: number; width?: number };
+
+export function Box(props: Props) {
+  return (
+    <div
+      style={{
+        backgroundColor: props.color ?? "lightblue",
+        height: props.height ?? 100,
+        width: props.width ?? 100,
```

Comment:
This is fine, but as mentioned in another comment I assume we're not going to vary the box sizes arbitrarily. As such, by making these not props, we can guarantee that every instance of Box is the same size.

Reply:
(Saw that you did this for practice -- sounds good for now though I think you'll get plenty of practice regardless!)